semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -214,6 +215,14 @@ def add_md_parser(subparsers):
         dest="md_enable_deprecated",
         action="store_false",
     )
+    parser.add_argument(
```

Comment:
This is a (preparation for a) major change IMHO, more than a "random editorial improvement". I like it though, it could be the first step towards better links, including from code generators.

Reply:
what's the concern here? The PR title?

Reply:
I changed the title, to align with the changelog entry. I think that resolves this.